addon/components/power-select-typeahead/trigger.js

```diff
@@ -45,48 +58,67 @@ export default Component.extend({
       } else {
         run.schedule('actions', null, newSelect.actions.open);
       }
-    } else if (!isBlank(newSelect.lastSearchedText) && newSelect.options.length === 0 && newSelect.loading) {
-      run.schedule('actions', null, newSelect.actions.close, null, true);
-    } else if (oldSelect.loading && !newSelect.loading && newSelect.options.length > 0) {
-      run.schedule('actions', null, newSelect.actions.open);
+    // } else if (!isBlank(newSelect.lastSearchedText) && newSelect.options.length === 0 && newSelect.loading) {
+    //   run.schedule('actions', null, newSelect.actions.close, null, true);
     }
+    // else if (oldSelect.loading && !newSelect.loading && newSelect.options.length > 0) {
+    //   run.schedule('actions', null, newSelect.actions.open);
+    // }
   },
```

Comment:
@cibernox I left these commented out for clarity.  So playing around with this, I can't see where removing this code leads to problems.  The result is the select box stays open while typing w/ an async search in progress.

Lmk what your thoughts are!